client/app/components/Input/InputTag.jsx

```diff
@@ -120,24 +155,19 @@ export class InputTag extends React.Component<Props, State> {
 
   displayAutocomplete = () => {
     const { placeholder } = this.props;
-    const { autocompleteLabel, checkboxes, autoHighlight } = this.state;
+    const { autocompleteLabel, checkboxes, autoHighlight, suggestions } = this.state;
     return (
-      <Autocomplete
-        getItemValue={this.getLabel}
-        items={checkboxes}
-        renderItem={this.renderItem}
-        shouldItemRender={this.shouldItemRender}
-        value={autocompleteLabel}
-        onChange={this.onAutocompleteChange}
-        onSelect={this.onSelect}
-        inputProps={{
-          className: `tagAutocomplete ${inputCss.tagAutocomplete}`,
-          onKeyPress: this.onKeyPress,
-          placeholder,
-        }}
-        wrapperStyle={{}}
-        renderMenu={this.renderMenu}
-        autoHighlight={autoHighlight}
+      <Autosuggest
+      suggestions={suggestions}
+      onSuggestionsFetchRequested={this.onSuggestionsFetchRequested}
+      onSuggestionsClearRequested={this.onSuggestionsClearRequested}
+      renderSuggestion={this.getSuggestionValue}
+      getSuggestionValue={this.renderSuggestion}
+      inputProps={{
```

Comment:
you need an `onChange` and a `value` property inside of `inputProps`. That's likely where the `cannot read property "trim" of undefined` is coming from 

https://github.com/moroshko/react-autosuggest/blob/master/src/Autosuggest.js#L491
https://github.com/moroshko/react-autosuggest/blob/master/src/Autosuggest.js#L491

Reply:
Yep, this definitely could be it.

**the following is from the docs**
_inputProps (required)
Autosuggest is a controlled component. **Therefore, you MUST pass at least a value and an onChange callback to the input**. You can pass any other props as well. For example:_

Reply:
I am currently working on this.

Right now I have:

            onChange: this.onAutocompleteChange,
            value : aututocompleteLabel,

inside of inputProps.